incidents/2021-11-15-alb-runs-out-of-nodes.md

```diff
@@ -0,0 +1,96 @@
+# 2021-11-15: ALB runs out of functioning nodes
+
+## description
+
+The ALB removed nodes from it's Target Group until only one remained and then that remaining node ran out of credits 
+after ~3 days. 
+
+## timeline
+
+UTC/ACDT
+
+03:16/13:46 - New Relic reports "Error Percentage (High)"
+
+* I click the acknowledge button and go investigating any spikes in traffic.
+    - these errors are typically enthusiastic bots, or a confluence of bots
+* I'm not logged in to NewRelic so it was never acknowledged.
+* traffic seems normal. I block a couple of random bots that are iterating through the corpus just in case with no effect.
+
+03:19/13:49 - New Relic reports "Journal (bypass CDN)"
+
+* I acknowledge again and log in to NewRelic this time.
+    - these are typically more serious
+    - I try to load the prod--journal.elifesciences.org/status in browser, no luck
```

Comment:
I didn't know about that status path. Will add that to my notes..

Reply:
Ha, just noticed that it's the old logo also! Anybody up for creating an ASCII art version of the new logo!? 😄